sklearn/covariance/_shrunk_covariance.py

```diff
@@ -563,6 +563,11 @@ class LedoitWolf(EmpiricalCovariance):
            [0.1616..., 0.8022...]])
     >>> cov.location_
     array([ 0.0595... , -0.0075...])
+
+    * See also :ref:`sphx_glr_auto_examples_covariance_plot_covariance_estimation.py`
+    for an example on how to fit a :class:`LedoitWolf` object to data and
+    for visualizing the performances of the Ledoit-Wolf estimator in
+    terms of likelihood.
```

Comment:
```suggestion
    terms of likelihood.
    
```

And it seems you also have to add a blank like here.

The same applies for the OAS.

Reason: CI shows the following errors:


    sklearn/covariance/_shrunk_covariance.py:docstring of sklearn.covariance._shrunk_covariance.LedoitWolf:117: WARNING: Bullet list ends without a blank line; unexpected unindent.
    sklearn/covariance/_shrunk_covariance.py:docstring of sklearn.covariance._shrunk_covariance.OAS:117: WARNING: Bullet list ends without a blank line; unexpected unindent. 

It pretty strictly follows some rules that you can look into [here](https://numpydoc.readthedocs.io/en/latest/format.html).

Reply:
Perfect! Thanks a lot again